request_a_govuk_domain/celery.py

```diff
@@ -22,4 +22,8 @@
         "task": "request_a_govuk_domain.request.tasks.check_email_failure_and_notify",
         "schedule": crontab(minute="*/1"),
     },
+    "application-status-checker": {
+        "task": "request_a_govuk_domain.request.tasks.check_application_status",
+        "schedule": crontab(hour="0"),
```

Comment:
> I think you need to set the minute=0 otherwise it will run every minute at 0 hrs

hour=0, doing it once a day at midnight. But will ask Robert on Monday.